manual/applicant.md

```diff
@@ -0,0 +1,95 @@
+
+## How to apply for access rights
+
+You can find all the resources you can apply for access from the Catalogue tab when you are logged in. The Catalogue tab also shows your unfinished applications. Continue filling in an application by selecting **View**.
+
+1. Start the application process by adding one or more resources to your shopping cart.
+If the resources have the same application form, they will be bundled together automatically. This means that you can apply for access rights to several resources with just one application form.
+![Bundle resources](img/bundle.png)
+2. Select **Apply** to continue to the application form.
+3. Follow the instructions on the application form and fill in the required fields. If you have questions regarding a specific application form, please contact the resource owner.
+4. If you are a part of a research group and other group members also need access rights, remember to invite them to your application by selecting **Invite member**.
+5. Accept the terms of use. Licenses define the terms and conditions for the use of the resource.
+6. To submit your application, select **Send application** after you have filled in all the necessary fields.
+
+:exclamation: Note! Once you send the application, you cannot edit it anymore.
+However, removing members from the application is always possible.
+
+Deleting sent applications is not possible because we want to always provide a full history of the application process.
+
+:point_up: Pro tip: If you want to use one of the old applications as a template for a new application, select **Copy as new**. Note that if your old application has attachments, they will also be copied.
+
+### Inviting members
+
+You can invite other people to join your application by selecting **Invite member** in the application form and providing their names and email addresses. This way all the research group members can apply for access rights to resources with just one form.
+
+Members get an email notification of the invitation immediately after you add them to your application. They can follow the link in the email to log in to REMS and accept the license terms. Members are granted access rights if the application is approved and they have accepted the license terms.
+
+## Approval process
+
+After you have sent the application, it will be forwarded to the application handlers who are in charge of granting the access rights. Approval process, as well as the processing time, both depend on the resource’s owner and application’s handlers.
+
+During the approval process, a handler can:
+
+- Approve or reject your application
+- Ask for additional information and return your application to you
+- Close your application and cancel the application process
+- Comment your application
+
+You can follow the approval process and read the possible comments from the application’s Events-section.
+
+![Event list](img/events.png)
```

Comment:
This image should probably be recreated, the event list has been re-styled

Reply:
Almost all images should be updated so I'll leave it for later now.